src/main/java/emissary/core/IBaseDataObjectXmlCodecs.java

```diff
@@ -40,8 +70,10 @@
 import static emissary.core.constants.IbdoXmlElementNames.PRIORITY;
 import static emissary.core.constants.IbdoXmlElementNames.PROCESSING_ERROR;
 import static emissary.core.constants.IbdoXmlElementNames.SHA256_ATTRIBUTE_NAME;
```

Comment:
Let's make this move this constant into the IBaseDataObjectXmlCodecs class itself and rename it as `SHA256`.  Its value is "sha256", it's only used within this class, and it's neither an element name nor an attribute name (it's a value we sometimes assign to the "encoding" attribute).  No Javadoc comment needed for it, it's self-documenting.

Reply:
Along with these two attribute names, I am planning on moving `ENCODING_ATTRIBUTE_NAME` as well, since that is also only used in XmlCodecs, and I feel as it will be the only _attribute name_ remaining, it shouldn't be by itself in the XmlElementName class.